FindTown/FindTown/Presentation/AuthScene/Signup/Nickname/NicknameViewModel.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  SignUpNickNameViewModel.swift
+//  FindTown
+//
+//  Created by 김성훈 on 2022/12/29.
+//
+
+import Foundation
+
+import FindTownCore
+import FindTownUI
+import RxSwift
+import RxRelay
+
+protocol NicknameViewModelType {
+    func goToLocationAndYears()
+}
+
+enum NicknameStatus {
+    case none
+    case complete
+    case duplicate
+    case inSpecialChar
```

Comment:
```suggestion
    case includeSpecialChar
```
이렇게하면 네이밍이 조금 더 명확할 것 같아요..!